tests/Pregel/CMakeLists.txt

```diff
@@ -15,6 +16,9 @@ target_link_libraries(arango_tests_pregel
   PRIVATE
   arango_inspection
   gtest
+  arango_lightweight
```

Comment:
I'm not sure if it is a good idea to link this into arango_tests_pregel because arango_tests_pregel is linked into arangodbtests. Can you remove this here and add it below ater line 29 to link it into arangodbtests_pregel?

Reply:
Will test and verify if this works.